modules/EvseManager/SessionLog.cpp

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2021 Pionix GmbH and Contributors to EVerest
+#include "SessionLog.hpp"
+#include "everest/logging.hpp"
+#include "v2gMessage.hpp"
+#include <chrono>
+#include <date/date.h>
+#include <date/tz.h>
+
+#include <boost/algorithm/string.hpp>
+#include <fmt/core.h>
+
+namespace module {
+
+SessionLog session_log;
+
+std::string to_rfc3339(std::chrono::time_point<date::utc_clock> t) {
```

Comment:
everest-framework now has a Everest::Date::to_rfc3339 (and from_) function that you should use here